### ISSUE TITLE
Plugins: Add a resource that teaches people what plugins are and how to use them

### ISSUE THREAD

**Issue by ndiego** [WordPress/wordpress.org#305]
This issue was ported over from [WordPress Slack](https://wordpress.slack.com/archives/C04U953K77A/p1714482106463999).

> **"how to use plugins" tutorial:** i don't know if it should be on the top or the bottom (although my preference is for it to be on the top), but there needs to be a beginner-friendly tutorial on "what are plugins and how to install, use and update them". i vividly remember the first time i downloaded a plugin, i saw the zip file and unzipped it, and couldn't figure out how to install it on my site. i'm sure there are plenty of other users like me. plugins are not at all intuitive, which is fine, but that's why documentation is important.
> - it's surprising that the only tutorials available on this page is about adding your plugin or creating your plugin.

There is a documentation article all about how to [Manage Plugins](https://wordpress.org/documentation/article/manage-plugins/). 

Perhaps we can update the footer, in conjunction with #224 and #257, to include a CTA and link to that resource.  

| Current Footer |
|-|
|<img width="1211" alt="image" src="https://github.com/WordPress/wordpress.org/assets/4832319/ee5f64bd-1973-4b1b-b192-60a94b584e7c">|

**Comment by courtneyr-dev** [comment]
Could we repurpose content from Learn? Rather than duplicate similar topic/content, let's make this more sustainable from a maintenance perspective and leverage resources via Training to handle the information. 
https://github.com/WordPress/Learn/issues/503
https://github.com/WordPress/Learn/issues/696
https://github.com/WordPress/Learn/issues/407
https://github.com/WordPress/Learn/issues/2113

**Comment by ndiego** [comment]
> Could we repurpose content from Learn? Rather than duplicate similar topic/content, let's make this more sustainable from a maintenance perspective and leverage resources via Training to handle the information.

Yes, I don't envision any new content being made necessarily. Mainly just a CTA somewhere that links to existing info.

**Comment by westnz** [comment]
Latest video I recorded on plugins and will form part of the Beginner user learning pathway: https://www.youtube.com/watch?v=WooNmrNKmcM
